app/api/routes.py

```
@@ -265,99 +259,27 @@ def get_ocr_image(
+    status_code=status.HTTP_204_NO_CONTENT,
     responses={204: {"description": "성공입니다"}},
 )
-def patch_record_exchange(
-        rec_id: int,
-        payload: RecordExchangeUpdateList,
-        db: Session = Depends(get_db),
-        current_user: User = Depends(AuthTokenDep)
+def patch_record_route(
+    rec_id: int,
+    payload: RecordPatch,
+    db: Session = Depends(get_db),
```

Ruff (0.14.5)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)